dao/src/main/java/greencity/repository/impl/EventSearchRepoImpl.java

```diff
@@ -145,7 +145,7 @@ private Predicate getPredicate(FilterEventDto filterEventDto, Long userId, Root<
     private void addEventTimePredicate(EventTime eventTime, Root<Event> eventRoot, List<Predicate> predicates) {
         if (eventTime != null) {
             ListJoin<Event, EventDateLocation> datesJoin = eventRoot.join(Event_.dates, JoinType.LEFT);
-            if (eventTime == EventTime.FUTURE) {
+            if (eventTime == EventTime.UPCOMING) {
                 predicates.add(
                     criteriaBuilder.greaterThan(datesJoin.get(EventDateLocation_.FINISH_DATE), ZonedDateTime.now()));
             } else if (eventTime == EventTime.PAST) {
```

Comment:
Why do we need `else if` here? 
There are only 2 items in the `EventTime` enum
I think it is better to have 2 separate `if` here